pkgs/by-name/op/opentelemetry-cpp/package.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, gbenchmark
+, gtest
+, zlib
+, pkg-config
+, protobuf
+, abseil-cpp_202301
```

Comment:
Abseil is only needed for compatibility with old compilers and `protobuf`. The former is not necessary in current nixpkgs stdenvs and the latter propagates `abseil-cpp` itself.
```suggestion
```